testsupport/tiers/checks.go

```diff
@@ -355,6 +362,43 @@ func (a *baseextendedidlingTierChecks) GetClusterObjectChecks() []clusterObjects
 		idlers(518400, "dev", "stage"))
 }
 
+func commonToolchainLabelsNamespaceCheck() namespaceObjectsCheck {
+	return func(t *testing.T, ns *corev1.Namespace, _ *wait.MemberAwaitility, userName string) {
+		assertExpectedToolchainLabels(t, ns, userName)
+	}
+}
```

Comment:
why do we need this check when we verify the set of labels as part of the `WaitForNamespace` method?
https://github.com/codeready-toolchain/toolchain-e2e/blob/1222fd9e4a79ca6d7cbfa357e5d980aa97536ff7/testsupport/wait/member.go#L653-L665

Reply:
I've removed this check in https://github.com/codeready-toolchain/toolchain-e2e/commit/fca5ee35c5ec4ed8de9397607a70ab6f1eac5d23 . Let's see how it goes without that check.

 I remember facing some issues with the presence of the label that was updated with some delay by the new version of the operator during the migration tests. But double checking it now, I agree it doesn't seem to be needed ! 👍